web/404.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
```

Comment:
This looks prime for go http/template... We're reusing the headers and structure in each page, so we should just do it once and redefine the main content each page

Reply:
I'm fine with this not happening tbh